signing_service/signing_service/utils.py

```diff
@@ -112,5 +115,41 @@ def __call__(  # type: ignore
 
 class Notifier(object):
 
-    def send(self) -> None:
-        pass
+    def __init__(
+        self,
+        from_email_address: str,
+        from_email_password: str,
+        to_email_addresses: list,
+    ) -> None:
+        self.from_email_address = from_email_address
```

Comment:
You may also use `__slots__` like in `SigningService` to make it faster.

Reply:
Done